beacon-chain/sync/rate_limiter.go

```diff
@@ -3,19 +3,25 @@ package sync
 import (
 	"reflect"
 	"sync"
+	"time"
 
-	"github.com/kevinms/leakybucket-go"
 	"github.com/libp2p/go-libp2p/core/network"
 	"github.com/pkg/errors"
 	"github.com/prysmaticlabs/prysm/v3/beacon-chain/p2p"
 	p2ptypes "github.com/prysmaticlabs/prysm/v3/beacon-chain/p2p/types"
 	"github.com/prysmaticlabs/prysm/v3/cmd/beacon-chain/flags"
+	leakybucket "github.com/prysmaticlabs/prysm/v3/container/leaky-bucket"
 	"github.com/sirupsen/logrus"
 	"github.com/trailofbits/go-mutexasserts"
 )
 
 const defaultBurstLimit = 5
 
+const leakyBucketPeriod = 1 * time.Second
+
+// Only allow in 2 batches per minute.
+const blockBucketPeriod = 30 * leakyBucketPeriod
```

Comment:
```suggestion
// Only allow in 2 batches per minute.
const blockBucketPeriod = 30 * time.Second
```

Could you write it as 30s instead? If leakyBucketPeriod was changed then your comment would be invalid